lib/bin/supllib/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if(CONFIG_FLOAT)
+  if(CONFIG_FP_HARDABI)
+    set(float_dir hard-float)
+  else()
+    assert(0 "Unreachable code")
+  endif()
+else()
+  assert(0 "Soft float not supported")
+endif()
+
+assert(GCC_M_CPU "GCC_M_CPU must be set to find correct lib")
+
+set(lib_path lib/${GCC_M_CPU}/${float_dir})
+
+set(SUPL_CLIENT_LIB_PATH ${ZEPHYR_BASE}/../supl/${lib_path})
+
+if(NOT EXISTS ${SUPL_CLIENT_LIB_PATH})
+  message(WARNING "Cannot locate SUPL Client library, should be located in your NCS root/supl. "
+                  "Information about how to download the library can be found at www.nordicsemi.no")
```

Comment:
My bad

Reply:
I wasn't able to find this, when will it be available?

Also, how many binary variants will it have?

Reply:
I don't know the exact time it will be available, but only one binary variant, hard-float for nrf9160.
I can send you a copy of the .a file if you want it for testing purposes?